src/workloads/networking/CommitLatencySingleUpdate.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Single threaded updates to measure commit latency for various write concern.
```

Comment:
We don't have strong guidelines, but most of the other product-perf workloads are in `workloads/scale` and platforms-team has been in `workloads/networking`. I can make a case for `networking` here though - as long as David's in agreement with where it belongs it's fine.

Reply:
InsertRemove being under scale is funny too, but in any case I can't put a single threaded test under scale. My alternatives were to create a new directory called either `transactions` or `consistency`. You can pick.

Reply:
> I can't put a single threaded test under scale

lol..agreed...

> My alternatives were to create a new directory called either transactions or consistency. You can pick.

🤷‍♂️ no real opinion I'll let @dalyd comment there or leave in `networking` since it's not really wrong. Bikeshedding is fun.